tests/Organization.py

```diff
@@ -367,11 +390,9 @@ def testInviteUserAsNonOwner(self):
 
     def testCreateMigration(self):
         self.org = self.g.get_organization("sample-test-organisation")
-        self.assertTrue(
-            isinstance(
-                self.org.create_migration(["sample-repo"]), github.Migration.Migration
-            )
-        )
+        migration = self.org.create_migration(["sample-repo"])
+        self.assertTrue(isinstance(migration, github.Migration.Migration))
+        self.assertListKeyEqual(migration.repositories, lambda r: r.has_pages, [False])
```

Comment:
Nah, this is fine to leave as is, you don't need to check it in every place, and this test is already a little awkward.